src/index.html

```diff
@@ -6,359 +6,311 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>John Roberts' CV</title>
     <link rel="stylesheet" href="./css/index.scss" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body class="my-cv">
-    <header>
-      <a href="/">HOME</a
-      ><a
-        href="https://docs.wixstatic.com/ugd/84770f_a594f88e0dd34b7ca253388dd6f4c9bb.pdf"
-        target="_blank"
-        >DOWNLOAD MY FULL RESUME</a
-      >
-    </header>
-
-    <main>
-      <!-- Avatar & Cover Background -->
-      <div>
-        <h1>John Roberts<br /><span>BUSINESS EXPERT</span></h1>
-        <img
-          src="https://static.wixstatic.com/media/86f442ff0674441faadc2994b1b33e77.jpg/v1/crop/x_130,y_160,w_1320,h_1311/fill/w_496,h_488,al_c,q_80,usm_0.66_1.00_0.01/86f442ff0674441faadc2994b1b33e77.webp"
-          alt="John Roberts"
-        />
+  <body>
+    <div class="bgimg"></div>
+    <!-- Header Section-->
+    <div class="header">
+      <div class="headleft">
```

Comment:
headleft and headright divs might not be required here, you can just position the links.